.github/shared/src/swagger.js

```diff
@@ -38,19 +38,19 @@ export class Swagger {
   /** @type {Map<string, Swagger> | undefined} */
   #refs;
 
-  /** @type {SpecModel | undefined} backpointer to owning SpecModel */
-  #specModel;
+  /** @type {Tag | undefined} Tag that contains this Swagger */
+  #tag;
```

Comment:
Swagger already had a backpointer, albeit an unprecise one only pointing to SpecModel